src/app/events/InteractionCreate.ts

```diff
@@ -56,12 +56,6 @@ export default class implements DiscordEvent {
         const bountyId: string = DiscordUtils.getBountyIdFromEmbedMessage(message);
         let request: any;
 
-        const deferTimeout = setTimeout(async ()=> {
```

Comment:
Why are you removing it?

Reply:
Two reasons:
1. We need to bot to fail if there are some use-cases which are not handled properly. This will cover up the problem
2. The message on deferReply is not appropriate to all the use cases we are dealing with here

I would like to add it back if we are sure that we have handled all the errors properly (solved 1)